backend/danswer/db/chat.py

```diff
@@ -275,6 +312,20 @@ def get_chat_messages_by_sessions(
     return db_session.execute(stmt).scalars().all()
 
 
+def get_search_docs_for_chat_message(
```

Comment:
This is fine but I believe SQLAlchemy handles this for you if you just set the relationships correctly in the db models

Reply:
Another brain hiccup- thank you for pointing this one out